workshop/part_0/README.md

```diff
@@ -0,0 +1,66 @@
+Part 0
+===================
+
+### Install kubectl
```

Comment:
We forgot about this part :joy:

Reply:
Yep ! Perfect timing : I was just thinking about it xD